website/live-demo/src/App.js

```diff
@@ -0,0 +1,118 @@
+import React, { Component } from 'react';
+import './App.css';
+import { Editor } from './editor.component';
+import { executeCodegen } from 'graphql-code-generator';
```

Comment:
Can't, because the webpack config of `create-react-app` does not allow to import from outside the project dir. 
I'm using the yarn workspace to get a symlink.